content/en/guide/v10/typescript.md

```diff
@@ -194,52 +250,60 @@ export class Button extends Component {
 }
 ```
 
-You can restrict event handlers by adding a type annotation for `this` to the function signature as the first argument. This argument will be erased after transpilation.
+You can provide a more specific type than `EventTarget` by using [this parameters](https://www.typescriptlang.org/docs/handbook/functions.html#this-parameters):
 
 ```tsx
+import { h, Component } from 'preact';
+
 export class Button extends Component {
-  // Adding the this argument restricts binding
+  // Define `this parameters`
   handleClick(this: HTMLButtonElement, event: MouseEvent) {
-    event.preventDefault();
-    if (event.target instanceof HTMLElement) {
-      console.log(event.target.localName); // "button"
-    }
+    console.log(this.tagName); // "BUTTON"
   }
 
   render() {
-    return (
-      <button onClick={this.handleClick}>{this.props.children}</button>
-    );
```

Comment:
This `(` and `)` are unnecessary. Therefore, I removed it.

Reply:
This is the formatting that prettier applies by default and we have that on other pages of our docs